### ISSUE TITLE
Some changes

### ISSUE THREAD

**Issue by nazar1ua** [Dromich/-vhost-add#2]
I want to make some minor changes:
```sh
#!/bin/bash

if (( $EUID != 0 )); then
    echo "Please run as root!!!"
    exit
fi

if grep "#Include etc/extra/httpd-vhosts.conf"  /opt/lampp/etc/httpd.conf; then
echo "Place before adding a new virtual host, you should uncomment(remove # symbol) on this line  \"Include etc/extra/httpd-vhosts.conf\" in the file    in /opt/lampp/etc/httpd.conf"
exit
fi

read -p "Please enter a virtual domain name:" domain
echo "Added new virtual domain: $domain"

read -p "Use this directory as a DocumentRoot (type yes/no): " tDir
case $tDir in
"yes") myDir="$(pwd)";;
"y") myDir="$(pwd)";;
*) read -p "Enter DocumentRoot directory patch for new host: " myDir;;
esac

echo "DocumentRoot directory patch: $myDir"

# START

read -p "Use directory $myDir as a root site folder (type yes/no): " rDir
case $rDir in
"yes") rootDir="$myDir";;
"y") rootDir="$myDir";;
*) read -p "Enter root directory patch for new site: " rootDir;;
esac

echo "Site root directory patch: $rootDir"

# END

echo "Adding host config in /opt/lampp/etc/extra/httpd-vhosts.conf  ...."

echo "<VirtualHost *:80>" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "	ServerAdmin admin@$domain" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "	DocumentRoot \"$myDir\"" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "	ServerName $domain" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "	ServerAlias www.$domain" >>/opt/lampp/etc/extra/httpd-vhosts.conf

# START

echo "	ErrorLog \"logs/$domain-error_log\"" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "	CustomLog \"logs/$domain-access_log\" common" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "	<Directory "$rootDir">" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "		Options Indexes FollowSymLinks Includes ExecCGI" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "		AllowOverride All" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "		Require all granted" >>/opt/lampp/etc/extra/httpd-vhosts.conf
echo "	</Directory>" >>/opt/lampp/etc/extra/httpd-vhosts.conf

# END

echo "</VirtualHost>" >>/opt/lampp/etc/extra/httpd-vhosts.conf

echo "Adding host( $domain )  alias  to 127.0.0.1  /etc/hosts ....."
if grep $domain  /etc/hosts; then
echo "This domain was added in hosts file earlier"
else

# TAB INSTEAD OF SPACE

echo "127.0.0.1	$domain" >>/etc/hosts
fi

echo "Xampp restarting....."
pkexec env DISPLAY=$DISPLAY XAUTHORITY=$XAUTHORITY /opt/lampp/lampp restart

echo "All done"
```

Also it would be good to check the availability of "net-tools" package. Thanks.

**Comment by nazar1ua** [comment]
This is very necessary for sites on Laravel

**Comment by Dromich** [comment]
> This is very necessary for sites on Laravel

Fixed in ver 0.15
